bears/c_languages/GNUIndentBear.py

```diff
@@ -153,6 +153,8 @@ def create_arguments(filename, file, config_file,
             Any command line options the indent binary understands. They
             will be simply passed through.
         """
+        if not max_line_length:
+            max_line_length = sys.maxsize
```

Comment:
gnu indent is written in C, so please look what numbers fit into the program and ensure this works on all platforms.